packages/node_modules/cerebral/.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "presets": ["es2015", "react"],
-  "plugins": ["version-transform"],
+  "plugins": ["version-transform", "transform-decorators-legacy"],
```

Comment:
`transform-decorators-legacy` has to be added to devDependencies. 
At the moment we get the following error on Travis :
```
ReferenceError: Unknown plugin "transform-decorators-legacy" specified in "/home/travis/build/cerebral/cerebral/packages/node_modules/cerebral/.babelrc" at 1, attempted to resolve relative to "/home/travis/build/cerebral/cerebral/packages/node_modules/cerebral"
```

Reply:
Okay added missing devDepencies, I missed committing that.

Reply:
Okay @henri-hulski I added the missing deps i forgot to commit that earlier. not sure on the eslint warning i didn't really change anything on that front.

Reply:
Now we get
```
Error: Cannot find module 'core-js/es7/reflect'
```
and also some prettier linting errors.

Reply:
added devDeps for angular. should be good to run now.